app/src/main/java/cz/martykan/forecastie/activities/MainActivity.java

```diff
@@ -83,6 +85,7 @@
     TextView todayHumidity;
     TextView todaySunrise;
     TextView todaySunset;
+    TextView todayUvIndex;
```

Comment:
According to Codacy, you should make this private.

Reply:
Wondering if I should keep it this way since  all the others are declared the same way.

Reply:
I think you can just move all of them to the top of the class. It really is a very well-known best-practice to put your class variables at the top of the class.